sql/derived_tables/po_ongoing.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS folio_reporting.po_ongoing;
+
+-- Create a local table for ongoing details in purchase orders.
+CREATE TABLE folio_reporting.po_ongoing AS
+SELECT
+    po.id AS po_id,
+    json_extract_path_text(data, 'ongoing', 'interval')::INT AS po_ongoing_interval,
+    json_extract_path_text(data, 'ongoing', 'isSubscription') AS po_ongoing_is_subscription,
+    json_extract_path_text(data, 'ongoing', 'manualRenewal') AS po_ongoing_manual_renewal,
+    json_extract_path_text(data, 'ongoing', 'renewalDate')::DATE AS po_ongoing_renewal_date,
```

Comment:
Please check the data types: `isSubscription` and `manualRenewal` appear to be Boolean (SQL: `boolean`), and `renewalDate` is encoded as a full timestamp (SQL: `timestamp with time zone`) even though the data do not include times.  I think we should stay close to the source data types by default, except where that causes problems for reporting queries or other problems.

Reply:
In my first draft I had cast to these data types and then I felt to simplify and interpret so I changed it.  But you're right to stay as close as possible to the source data.